services/movie_session.py

```diff
@@ -3,6 +3,14 @@
 from db.models import MovieSession
 
 
+def get_taken_seats(movie_session_id: int) -> list:
+    return list(
+        MovieSession.objects.get(id=movie_session_id).tickets.values(
```

Comment:
Consider handling the `DoesNotExist` exception for `MovieSession.objects.get(id=movie_session_id)` to avoid runtime errors if the movie session with the given ID does not exist.